### PR TITLE
Update index+budget+bills

### PR DESCRIPTION
**Pull request by WeijingZeng**
Update index+budget+bills

### CODE REVIEW

File: data/budget.js [hunk 1]
```diff
@@ -4,6 +4,12 @@ const users = require("./users");
 const uuid = require('node-uuid');
 
 let exportedMethods = {
+    
+    getAllBudget() {
```
Comment (psharm8) [inline data/budget.js:8]: I don't think this method would serve any purpose.
At best we would need budgets by userId to update after expenses.

**Review by psharm8 — CHANGES_REQUESTED** [review]
getAllBudget is not needed.

File: data/bills.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+     getBillByCategory(category) {
+        return bill().then((billCollection) => {
+            return billCollection
+                .find({"category":  category})
+                .toArray();
+        });
+    },
+
+    //category and date could get from select on webpage
+    addBill(category, amount, date, note, userID ) {
```
Comment (anuragpatil94) [inline data/bills.js:51]: Should we add currency in bills?

Comment (psharm8) [reply data/bills.js:51]: That would mean more work... what if the person has a budget in INR and adds a bill in USD... getting latest exchange rates is going to be a nightmare :|

File: data/budget.js (hunk 1 above)
Comment (psharm8) [reply data/budget.js:8]: @WeijingZeng please remove this method and commit.

File: data/bills.js (hunk 2 above)
Comment (anuragpatil94) [reply data/bills.js:51]: hmm true

File: data/budget.js (hunk 1 above)
Comment (anuragpatil94) [reply data/budget.js:8]: I think this method should be for every user, as a user may have many budgets for eg: budget for categories.

**Pull request merged**